sdk/communication/communication-job-router/src/jobRouterAdministrationClient.ts

```diff
@@ -146,12 +142,11 @@ export class JobRouterAdministrationClient {
     this.client.pipeline.addPolicy(authPolicy);
   }
 
-  // Classification Policy Actions
   /**
    * Creates a classification policy.
    * Returns the created classification policy.
-   * @param classificationPolicyId - Id of the classification policy.
-   * @param options - Create classification options.
+   * @param classificationPolicyId - id of the classification policy.
+   * @param options - Options for creating a classification policy. Uses merge-patch semantics: https://datatracker.ietf.org/doc/html/rfc7386.
```

Comment:
nit: don't need this for create, only updates

Reply:
Keeping for now, @sarkar-rajarshi suggested we have these on creates too.